src/pages/sicp/subcomponents/chatbot/ChatBox.tsx

```diff
@@ -42,6 +48,10 @@ const ChatBox: React.FC<Props> = ({ getSection, getText }) => {
     if (userInput.trim() === '') {
       return;
     }
+    if (userInput.length > maxContentSize) {
+      setMessages(prev => [...prev, MESSAGE_TOO_LONG_MESSAGE]);
+      return;
+    }
```

Comment:
I don't think we should error when the user submits. Instead, the input field should not allow more than `maxContentSize` characters.

Reply:
This can be a simple indicator of `maxContentSize - userInput.length` and set the `onChange` to ignore extra characters beyond `maxContentSize`

Reply:
Thanks for the comment. Feature added.